v3/cypress/e2e/graph.spec.ts

```diff
@@ -47,6 +43,32 @@ context("Graph UI", () => {
     cy.wait(2500)
   })
   describe("graph view", () => {
+    it.skip("should highlight a selected graph point", () => {
+      // This test is the outcome of a SPIKE to explore testing graph interactions.
+      // It partially validates interactions but requires further PIXIJS-level support.
+      // https://github.com/concord-consortium/codap/pull/1637
+
+      // Select the target table cell
+      table.getGridCell(2, 2).should("contain", "African Elephant").click({ force: true })
+
+      // Verify the graph's component title matches the collection name
+      c.getComponentTitle("graph").should("contain", collectionName)
+
+      // Re-click the table cell to ensure interaction consistency
+      table.getGridCell(2, 2).click({ force: true })
+
+      // Future goal: Validate the highlighted graph point
+      // ChatGPT suggests this approach could work if PIXIJS exposes DOM elements
+      // or provides API/event hooks that allow direct verification of point states.
+      // cy.get('[data-testid="graph"] canvas')
+      //   .should('be.visible') // Ensure the canvas is rendered
+      // cy.get('[data-testid="graph"]')
+      //   .find('svg .below-points-group circle') // Intended to locate graph points
```

Comment:
This won't work because there are no DOM elements that correspond to the points, SVG or otherwise.